src/sdk/conference/src/main/java/owt/conference/RemoteStream.java

```diff
@@ -21,7 +21,7 @@
  * RemoteStream represent the stream published by other endpoints in the conference.
  */
 public class RemoteStream extends owt.base.RemoteStream {
-    public SubscriptionCapabilities subscriptionCapability;
+    public SubscriptionCapabilities extraSubscriptionCapability;
```

Comment:
Please add description for `extraSubscriptionCapability`. It's a public API.

It's better to align with other SDKs, i.e.: `extraCapabilities` and `settings`. But it can be done in another PR.